### PR TITLE
Add app integration test commands

### PR DESCRIPTION
**Pull request by wessmith**
## Overview

Adds a set of commands to facilitate automatic verification of a ButtonMerchant integration

### CODE REVIEW

File: Source/AppIntegrationVerification.swift [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+
+import UIKit
+
+internal class AppIntegrationVerification: NSObject {
+    
+    static let shared = AppIntegrationVerification()
+    
+    enum Command: String {
+        
+        static let testHost = "test.bttn.io"
```
Comment (chrismaddern) [inline Source/AppIntegrationVerification.swift:33]: We should make sure this is registered 😆

File: Source/AppIntegrationVerification.swift [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+            return url.path.components(separatedBy: "/").last
+        }
+        
+        return nil
+    }
+    
+    static func isEchoCommand(url: URL) -> Bool {
+        guard let urlComponents = URLComponents(string: url.absoluteString),
+              let echoItem = urlComponents.queryItems?.first(where: { $0.name == Self.echo.rawValue }),
+              ((echoItem.value as NSString?)?.boolValue ?? false) else {
```
Comment (najmsheikh) [inline Source/AppIntegrationVerification.swift:141]: So many ?????? 😆

**Review by najmsheikh — APPROVED** [review]
The copyright headers _really_ add to the line count hah

File: Source/AppIntegrationVerification.swift (hunk 2 above)
Comment (wessmith) [reply Source/AppIntegrationVerification.swift:141]: It _might_ be an `NSString`, that _might_ be a `Bool` that should otherwise be `false`

**Review by chrismaddern — APPROVED** [review]

**Pull request merged**